Marlin/src/MarlinCore.cpp

```diff
@@ -1217,6 +1217,11 @@ void setup() {
     SETUP_RUN(password.lock_machine());      // Will not proceed until correct password provided
   #endif
 
+  // Validate DELAY_CYCLES_ITERATION_COST
+  #if ENABLED(MARLIN_DEV_MODE) && __CORTEX_M != 7 && (defined(__arm__) || defined(__thumb__))
```

Comment:
Here CORTEX WITHIN 4, 7 since cost is not needed when debug timer is used

Reply:
I removed the validate